cucumber.conf.js

```diff
@@ -0,0 +1,35 @@
+const {
+  Before,
+  BeforeAll,
+  AfterAll,
+  After,
+  setDefaultTimeout,
+} = require("@cucumber/cucumber");
+const { chromium } = require("playwright");
+const { config } = require("./config");
+const {
+  deleteFolder,
+} = require("./tests/cucumber/acceptance/helpers/HttpHelpers");
+
+setDefaultTimeout(60000);
+BeforeAll(async function () {
+  global.browser = await chromium.launch({
+    headless: config.HEADLESS,
+    slowMo: config.SLOW_MO,
+  });
+});
+
+AfterAll(async function () {
+  await deleteFolder();
```

Comment:
cleanup in `AfterAll`? or do you want to clean up folders after each scenario?
```suggestion
AfterAll(async function () {
  await deleteFolder();
```